R/variantSetTests.R

```diff
@@ -58,14 +58,14 @@ testVariantSet <- function( nullmod, G, weights,
     }else{
         burden <- colSums(t(G) * weights)
     }
-    
+
     # adjust burden for covariates and random effects
     if (burden.test == "Score") {
         Gtilde <- calcGtilde(nullmod, burden)
         if(is.null(nullmod$RSS0)){
```

Comment:
@amstilp again - i think this could be moved to your update old null models function